src/ixdat/techniques/ms.py

```diff
@@ -348,41 +348,74 @@ def as_mass(self, item):
             return self.as_mass(new_item)
         raise TypeError(f"{self} does not recognize '{item}' as a mass.")
 
-    def gas_flux_calibration(self, mol, mass, tspan, chip=None):
-        """Simple pure-gas flux calibration, using external MS quantification package
+    def gas_flux_calibration(
```

Comment:
Would it make sense to add a "warning" if one has activated a plugin but mistakenly useds this function instead of sig_gas_flux_calibration? Or should this function simply just pipe through to external quantification packages in case those af active?

Reply:
Kenneth, Soren and me had a long discussion on why we don't want it to just "pipe through" to the external package (in a nutshell: because it makes it very hard to understand what's going on if you dont really know what you're doing). I think it's a good idea with a warning though, especially since at the moment it just goes to the external package and this change will be a bit confusing for existing users

Reply:
I've put a warning here. It would make sense to put it in `gas_flux_calibration_curve` as well but I can't as it is now since `siq_gas_flux_calibration_curve` calls `gas_flux_calibration_curve`. Not sure if that inconsistency just makes it worse?

Reply:
Can `gas_flux_calibration_curve` figure out that it's being called from `siq_gas_flux_calibration_curve` and disable a warning in that case? Or does it become too complex then?
I dont think that the inconsistency, but we need to make sure that the documentation is good enough.

Reply:
Or another way would be to place the code of `gas_flux_calibration_curve()` in and internal function of the class  `_gas_flux_calibration_curve()`. 

```
gas_flux_calibration_curve():
     if external_package_active:
        warn.warning('You have activated an external package. Call this function instead 'siq_gas_flux_calibration_curve')

    _gas_flux_calibration()
```